azure-kusto-data/example.js

```diff
@@ -6,39 +6,43 @@ const KustoConnectionStringBuilder = require("azure-kusto-data").KustoConnection
 const ClientRequestProperties = require("azure-kusto-data").ClientRequestProperties;
 const uuidv4 = require("uuid/v4");
 
-let clusterName = "";
-let username = "username";
-let password = "password";
+const clusterConectionString = "https://<cluster>.<region>.kusto.windows.net";
+const database = "<databaseName>";
+const table = "<tableName>";
 
-const kcs = KustoConnectionStringBuilder.withAadUserPasswordAuthentication(`https://${clusterName}.kusto.windows.net`, username, password);
+const kcs = KustoConnectionStringBuilder.withAadDeviceAuthentication(clusterConectionString);
 const kustoClient = new KustoClient(kcs);
+start();
 
-kustoClient.execute("db", "TableName | limit 1", (err, results) => {
-    if (err) throw new Error(err);
-    console.log(JSON.stringify(results));
-    console.log(results.primaryResults[0].toString());
-});
+async function start() {
+    try {
+        const results = await kustoClient.execute(database, `['${table}'] | limit 1`);
+        console.log(JSON.stringify(results));
+        console.log(results.primaryResults[0].toString());
+    }
+    catch (error) {
+        console.log(error);
+    }
 
-// providing ClientRequestProperties
-// for a complete list of ClientRequestProperties
-// go to https://docs.microsoft.com/en-us/azure/kusto/api/netfx/request-properties#list-of-clientrequestproperties
-let clientRequestProps = new ClientRequestProperties();
-const ONE_MINUTE = 1000 * 60;
-clientRequestProps.setTimeout(ONE_MINUTE);
+    // providing ClientRequestProperties
+    // for a complete list of ClientRequestProperties
+    // go to https://docs.microsoft.com/en-us/azure/kusto/api/netfx/request-properties#list-of-clientrequestproperties
+    let clientRequestProps = new ClientRequestProperties();
+    const ONE_MINUTE = 1000 * 60;
```

Comment:
You have 2 different naming conventions for consts in this file... Consider aligning with 1